gnomad/variant_qc/evaluation.py

```diff
@@ -20,8 +20,8 @@ def compute_ranked_bin(
     n_bins: int = 100,
     desc: bool = True,
 ) -> hl.Table:
-    """
-    Returns a table with a bin for each row based on the ranking of `score_expr`.
+    r"""
```

Comment:
Is a raw string necessary here?

Reply:
It complained about it because of the `\` needed below for formatting the table